src/device-status/device_status.py

```diff
@@ -26,7 +26,7 @@ def get_all_devices(tenant):
     model = Device(tenant)
     results = model.get_devices()
     return [
-        device for device in results if device.get("isActive") or (device.get("mobility") and device.get("powerType") and device.get("network", "") == "airqo")
+        device for device in results if (device.get("isActive") or (device.get("mobility") and device.get("powerType"))) and device.get("network", "") == "airqo"
```

Comment:
only use `active `devices (mostly static) or mobile devices that have a `power type` AND all MUST belong to the AirQo network. cc: @sserurich @deogratius2phantom @NoahNsimbe